### ISSUE TITLE
Stylus blocks are treated as CSS

### ISSUE THREAD

**Issue by JulianWels** [sveltejs/language-tools#590]
Hey! :wave:

**Describe the bug**
I want to use stylus in my svelte components. Syntax highlighting is correct, but it seems the code is still treated as CSS. The code-completion is confused and I get a lot of errors regarding braces, like `css(css-lcurlyexpected)`.

I followed [this guide](https://github.com/sveltejs/language-tools/blob/f5dd7d65464032929c4bb363cabf81ce4088c2fd/docs/preprocessors/in-general.md) but that didn't help and neither did disabling everything style-related in VS Code and the language tools.

It seems to me that the language tools [don't recognize stylus](https://github.com/sveltejs/language-tools/blob/a14cf1b61b5958159e342f96a391b9ac94eb5aa1/packages/language-server/src/plugins/css/service.ts#L31-L33) as a language yet.

**To Reproduce**
1. Install "Svelte for VS Code" extension (svelte.svelte-vscode)
2. Install "language-stylus" extension (sysoev.language-stylus)
3. Create new svelte file with `<style lang="stylus">` tag

For example:
```svelte
<style lang="stylus">
  div
    background #FFFFFF
    width 290px
</style>
```

**Expected behavior**
- No errors or stylus related errors.
- No syntax completion or stylus syntax completion.

I don't necessarily need all language features within this block (although it would be nice), but the css errors make working with svelte and stylus really hard.

**Screenshots**
![Screenshot of Problem](https://i.imgur.com/6adta0t.png)

**System (please complete the following information):**
 - OS: macOS
 - IDE: VSCode
 - Plugin/Package: sveltePreprocess and the others mentioned above

**Comment by dummdidumm** [comment]
Did you set [this setting](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#svelteplugincssdiagnostics) to `false`? Doing so should turn off these diagnostics.

**Comment by JulianWels** [comment]
Oh yes, that at least disabled diagnostics – Thanks!!

But vs code still seems to think it's CSS and the autocompletion goes haywire. I assume because the [fallback is always CSS](https://github.com/sveltejs/language-tools/blob/a14cf1b61b5958159e342f96a391b9ac94eb5aa1/packages/language-server/src/plugins/css/service.ts#L47)?

**Comment by dummdidumm** [comment]
That's correct. We need to skip these things if `lang="stylus"` is set. In the meantime, you can disable the feature through [this setting](https://github.com/sveltejs/language-tools/tree/master/packages/svelte-vscode#svelteplugincsscompletions).

**Comment by JulianWels** [comment]
Yay, that did the trick! 🎉 

I don't know if you want to close this. IMHO some support for stylus would be nice, because [svelte-preprocess supports it](https://github.com/sveltejs/svelte-preprocess/blob/master/docs/preprocessing.md#stylus).
But looking at [veturs code](https://github.com/vuejs/vetur/tree/60378c3f68a94a4ae8b80607e3f321cfa3476a26/server/src/modes/style/stylus), it seems adding full support for stylus might be comparatively complicated.

But maybe it's possible to at least disable CSS language features, by as you said, adding ['stylus' here](https://github.com/sveltejs/language-tools/blob/096ee6997b031ae109e4b9864c815a55607728a9/packages/language-server/src/plugins/css/CSSPlugin.ts#L285) and add [a check here](https://github.com/sveltejs/language-tools/blob/master/packages/language-server/src/plugins/css/CSSPlugin.ts#L133) as well, like:
```js
if (shouldExcludeCompletion(kind)) {
    return null;
}
```

----

**TL;DR for humans that get CSS errors when using stylus in VS Code:**
```json
"svelte.plugin.css.completions.enable": false
"svelte.plugin.css.diagnostics.enable": false
```

**Comment by dummdidumm** [comment]
Diagnostics and autocompletion is now turned off when `lang="stylus"`.
